test-case/manual/check-fail-no-tplg.md

```diff
@@ -0,0 +1,22 @@
+# No sof tplg test
+Ensure system not hang when missing sof-tplg
+
+# Preconditions
+PulseAudio should be enabled
+
+# Test Description
+* Remove sof-tplg file to let sof probe fail
+
+## Test Case:

```

Comment:
Same question here again --- do you intend for the manual test to be executed twice or are you testing two similar but slightly different cases? If its just one case executed twice, lets remove the duplication of instructions and just say to run it twice. If its two slightly different cases, lets draw attention to that and make that clearer.

- Instructions 1-5 / Case 1:
  - system is on
  - rename sof-tplg dir
  - reboot
  - check that missing sof-tplg error is in dmesg
  - test s/r cycle
  - restore sof-tplg dir
  - reboot
  - sof-tplg loads properly
<br></br>
- Instructions 6-8 / Case 2 (same as case 1, just no reboot immediately after sof-tplg dir is renamed): 
  - system is on
  - rename sof-tplg dir
  - ~reboot~
  - check that missing sof-tplg error is reported in dmesg
  - test s/r cycle
  - restore sof-tplg dir
  - reboot
  - sof-tplg loads properly